components/Tabs/src/TabBarView/MDCTabBarItemCustomViewing.h

```diff
@@ -28,4 +29,7 @@
 @property(nullable, nonatomic, strong)
     UIView<MDCTabBarSelectionIndicatorSupporting> *mdc_customView;
 
+/** The ripple contronller to display the ripple touch effect. */
+@property(nullable, nonatomic, strong) MDCRippleTouchController *rippleTouchController;
```

Comment:
Let's not force a ripple touch controller on the view. Instead, clients should be allowed to integrate Ripple however it works best for their custom views.

Reply:
Done.